LockboxXCUITests/LockBoxScreenGraph.swift

```diff
@@ -191,6 +207,62 @@ extension BaseTestCase {
         navigator.performAction(Action.FxATypePassword)
     }
 
+    func loginFxAccount() {
+        userState.fxaPassword = passwordTestAccountLogins
+        userState.fxaUsername = "test-b62feb2ed6@restmail.net"
+        navigator.goto(Screen.FxASigninScreenEmail)
+        waitforExistence(app.navigationBars["Lockbox.FxAView"])
+        waitforExistence(app.webViews.textFields["Email"], timeout: 10)
+        navigator.performAction(Action.FxATypeEmail)
+        waitforExistence(app.webViews.secureTextFields["Password"])
+        navigator.performAction(Action.FxATypePassword)
+    }
+
+    func skipAutofillConfiguration() {
```

Comment:
In this case, and for the comment below about tapOnFinishButton(), initially I created the actions/screens on the graph. But I'm using this method instead of the graph because depending on the iOS version the autofill screen will be shown or not. I was not able to manage that variable to route from the graph to one screen or another depending on that. If I use the graph the tests will only work on iOS 12. 
Not sure if that's doable using the graph...